multiversx_sdk_core/transaction_factories/delegation_transactions_factory_test.py

```diff
@@ -252,3 +252,85 @@ def test_create_transaction_for_setting_metadata(self):
         assert transaction.data
         assert transaction.data.decode() == "setMetaData@6e616d65@77656273697465@6964656e746966696572"
         assert transaction.amount == 0
+
+    def test_create_transaction_for_delegating(self):
+        sender = Address.new_from_bech32("erd18s6a06ktr2v6fgxv4ffhauxvptssnaqlds45qgsrucemlwc8rawq553rt2")
+        delegation_contract = Address.new_from_bech32("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtllllls002zgc")
```

Comment:
suggestion for another PR, extract these addresses to vars/constants and reuse them in these tests

Reply:
Indeed. Will do so in the next PR.